README.md

```diff
@@ -19,20 +19,6 @@
     </a>
 </div>
 
-## Table of contents
-   * [Overview](https://github.com/openscilab/nava#overview)
-   * [Installation](https://github.com/openscilab/nava#installation)
-   * [Usage](https://github.com/openscilab/nava#usage)
-   * [Engine](https://github.com/openscilab/nava#engine)
-   * [Issues & Bug Reports](https://github.com/openscilab/nava#issues--bug-reports)
-   * [Contribution](https://github.com/openscilab/nava/blob/main/.github/CONTRIBUTING.md)
-   * [Authors](https://github.com/openscilab/nava/blob/main/AUTHORS.md)
-   * [License](https://github.com/openscilab/nava/blob/main/LICENSE)
-   * [Show Your Support](https://github.com/openscilab/nava#show-your-support)
-   * [Changelog](https://github.com/openscilab/nava/blob/main/CHANGELOG.md)
-   * [Code of Conduct](https://github.com/openscilab/nava/blob/main/.github/CODE_OF_CONDUCT.md)
-
-## Overview
```

Comment:
Why did you delete this line?

Reply:
I thought it’s redundant now. Since we are starting with overview anyways, there is no need to mention the title again.

Reply:
I suggest keeping the `## Overview` header to properly separate the badge and overview sections.